scripts/add-domains.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+if (process.argv.includes('--help') || process.argv.includes('-h')) {
+    console.info(`Usage: ${__filename.slice(__dirname.length + 1)} [-h|--help] [--dry]`);
+    process.exit(0);
+}
+
+const isDry = process.argv.includes('--dry');
+
+const fs = require('fs');
+
+process.env.ENABLE_PROVIDER_DOMAINS = 'true';
+const config = require('../config/environment')(process.env.EMBER_ENV);
+
+const {providers} = config.PREPRINTS;
+const hostsFileName = '/etc/hosts';
+const hostIP = '127.0.0.1';
+const hostsFile = fs.readFileSync(hostsFileName, {encoding: 'utf8'});
+const sectionHeader = '## EMBER-PREPRINTS ##\n';
+const sectionFooter = '\n## /EMBER-PREPRINTS ##';
+const rgx = new RegExp(`(?:${sectionHeader})(.|\\s)*(?:${sectionFooter})`, 'm');
+const domainProviders = providers
+    .slice(1)
```

Comment:
I was trying to avoid modifying the original config too much, which would have downstream changes--not ideal, but neither is creating a ton of merge conflicts if it can be avoided.